hypha/apply/funds/templates/funds/application_base.html

```diff
@@ -1,27 +1,26 @@
-{% extends "base.html" %}
-
-{% load wagtailcore_tags static i18n util_tags %}
-{# Dont include fixed apply button on this page #}
-{% block apply_button %}{% endblock %}
-{% block header_modifier %}header--light-bg{% endblock %}
+{% extends "base-apply.html" %}
+{% load wagtailcore_tags static i18n util_tags heroicons %}
+{% block body_class %}{% endblock %}
 
 {% block content %}
-{% if form.errors or form.non_field_errors %}
-<div class="wrapper wrapper--medium wrapper--error">
-    <svg class="icon icon--error"><use xlink:href="#error"></use></svg>
-    {% if form.non_field_errors %}
-        <ul>
-            {% for error in form.non_field_errors %}
-                <li class="error">{{ error }}</li>
-            {% endfor %}
-        </ul>
-    {% else %}
-    <h5 class="heading heading--no-margin heading--regular">{% trans "There were some errors with your form. Please amend the fields highlighted below" %}</h5>
+<h1 class="mt-8">{{ page.title }}</h1>
+
+<div class="wrapper wrapper--form">
```

Comment:
I think the "wrapper--medium" class is better to keep here.

Reply:
Let me give that a try and see how that looks.

Reply:
Updated. 
![Screenshot 2023-07-20 at 06 32 27](https://github.com/HyphaApp/hypha/assets/236356/7334643e-ad05-4579-85a1-d97173ca995b)